metals/src/main/scala/scala/meta/internal/metals/ConfiguredLanguageClient.scala

```diff
@@ -85,7 +92,7 @@ final class ConfiguredLanguageClient(
   override def metalsExecuteClientCommand(
       params: ExecuteCommandParams
   ): Unit = {
-    if (config.executeClientCommand.isOn || clientCapabilities.executeClientCommandProvider) {
+    if (config.executeClientCommand.isOn || clientCapabilities.executeClientCommandProvider || initializationOptions.executeClientCommandProvider) {
```

Comment:
In a lot of these boolean checks can you pull them out into a variable like you did below? For example this one being like `isExecuteClientCommandProvider` or something like that. Now that we have three different boolean checks in here, it's getting a bit messy.